schemas/blocks/confirmation_question.json

```diff
@@ -40,6 +45,27 @@
       "id",
       "type",
       "questions"
+    ],
+    "oneOf": [
```

Comment:
Why does this not apply to all blocks?

Reply:
Because interstitial doesn't have `questions`?

Reply:
Ah, it does

Reply:
We should define this as a common type and reuse the same definition

Reply:
This is seemingly difficult to extract with current versions of json schema.